src/main/groovy/com/base2/rest/RestApiJobManagement.groovy

```diff
@@ -44,19 +53,45 @@ class RestApiJobManagement extends MockJobManagement {
     createOrUpdateConfig(viewName, config, ignoreExisting, true)
   }
 
+  private String getJobDescription(xml){
+    try {
+      def xmlObj = new XmlSlurper().parseText(xml)
+      return xmlObj.description
+    }catch (Exception ex){
+      println "Error reading description from remote xml"
+      return ""
+    }
+  }
+
   boolean createOrUpdateConfig(String name, String xml, boolean ignoreExisting, boolean isView) throws NameNotProvidedException {
     boolean success
     String status
 
     String existingXml = fetchExistingXml(name, isView)
+    def update = {
+      success = update(name, xml, isView)
+      status = success ? 'updated' : 'update failed'
+    }
     if (existingXml) {
-      if (ignoreExisting) {
+      //check if job description is any of dirty markers
+      def jobDescription = getJobDescription(existingXml)
+      if(__DIRTY_MARKERS.contains(jobDescription.toUpperCase())){
```

Comment:
@aaronwalker check is already looking up whether job description is in list of predefined ones (`__DIRTY_MARKERS`) - line 78